### PR TITLE
Proposals/section spatial units

### PR DESCRIPTION
**Pull request by corymosiman12**
- see proposal
- implementation in schema added

### CODE REVIEW

File: BuildingSync.xsd [hunk 1]
```diff
@@ -13934,6 +13879,67 @@
...
+          </xs:restriction>
+        </xs:simpleType>
+      </xs:element>
+      <xs:element name="NumberOfUnits" minOccurs="0">
+        <xs:annotation>
+          <xs:documentation>Number of individual units within the premises.</xs:documentation>
+        </xs:annotation>
+        <xs:complexType>
+          <xs:simpleContent>
+            <xs:extension base="xs:integer">
```
Comment (macintoshpie) [inline BuildingSync.xsd:13913]: breaking change, but this should be a `postiveInteger`. Something to address in v3? Created ticket here: #301

Comment (corymosiman12) [reply BuildingSync.xsd:13913]: true. I think I will leave the current implementation to remain non-breaking, but started adding checklists to #301

**Comment by yan130** [comment]
Could I know this PR is for which UDF?

**Comment by corymosiman12** [comment]
Hi Yan - we have been trying to link all PRs back to the [UDFs spreadsheet](https://docs.google.com/spreadsheets/d/1LaztYTrT7_iPQTcTDEQZAB2fVJ23G3Wo/edit#gid=595543811) for reference. Let me know that you find it!

**Comment by nllong** [comment]
Hi Yan, can you review this proposal when you get a chance?

**Review by yan130 — APPROVED** [review]

**Comment by yan130** [comment]
This change will not have influence on AT's XML.

**Comment by nllong** [comment]
@JieXiong9119 -- can you dedupe the SpatialUnit complex name type?

Thanks!

**Comment by nllong** [comment]
> @JieXiong9119 -- can you dedupe the SpatialUnit complex name type?
> 
> Thanks!

Hey @JieXiong9119 -- I added `spatialunit` and `spatialunits` to the exclusion for the tests. We would need to create a breaking change to address this, so let's wait on that.

**Pull request merged**